docs/cudf/source/cudf_polars/api.md

```diff
@@ -0,0 +1,17 @@
+(cudf-polars-api)=
```

Comment:
Just checking: Did you build the docs locally, and verify that it looks right?

Reply:
Sorry that's vague. I meant that links work, formatting looks good, code samples render correctly, etc.

Reply:
Yeah. It's building without warnings, and the link to this section from `engine_options.md` is working as expected.

I wouldn't say the formatting of the type signature looks *good*, but it's as expected I think:

![image](https://github.com/user-attachments/assets/683018de-a309-4633-ae8a-f6e76781b0a0)